PolyEngine/Engine/Src/KeyBindings.hpp

```diff
@@ -1,11 +1,12 @@
 #pragma once
 
+#if defined(_WIN32)
 #include <Windows.h>
 
 namespace Poly
 {
-	enum class eKey
-	{	
+	enum class eKey : uint8_t
```

Comment:
I actually checked and all our values are within range. As far as I know, there are no keycodes higher than 255 in the wild (although both WinAPI and X11 use an int for storing them, so I should probably change it).